docs/FAQ.md

```diff
@@ -132,6 +133,21 @@ code-server --install-extension ms-python.python
 code-server --install-extension downloaded-ms-python.python.vsix
 ```
 
+### Installing VSIX Extensions via the Command Line
```

Comment:
I think this overlaps a little with the previous paragraph that lists how to install extensions on the CLI.

What if we move that into this heading, then create another heading for this content with something like "Installing OpenVSX extensions via the command line" for this content?

We could also mention they can switch early to OpenVSX by setting `SERVICE_URL` and `ITEM_URL` (we already have a section so maybe we just link it from here).

Reply:
Actually maybe we should put this under "How can I request a missing extension?"

Reply:
Agreed.

Yes! Let's do that!

Reply:
Or just remove that section.

Reply:
I opted to add it under "How can I request a missing extension" - I say we move remove the extension request section once we fully switch to OpenVSX

Reply:
My thinking is that the flow from a user might look like:

"How do I install an extension?" 
"Oh, it's not available, how do I request it?"
"Ah, they aren't accepting requests, how do I install it then?"
"OK, I can either do it manually or download from Open VSX."

Reply:
Ahh yeah good point about that. Okay I'll make these changes then re-request a review! Thanks for adding the structure/flow too! That helps a ton!!

Reply:
Embarrassingly, I didn't know you could do it this way, hence the `curl` suggestion:
```
SERVICE_URL=https://open-vsx.org/vscode/gallery ITEM_URL=https://open-vsx.org/vscode/item code-server --install-extension stylelint.vscode-stylelint
```